hanadb_exporter/db_manager.py

```diff
@@ -87,10 +92,11 @@ def _get_connection_data(self, userkey, user, password):
             raise ValueError(
                 'Provided user data is not valid. userkey or user/password pair must be provided')
 
-        return {'userkey': userkey,
-                'user': user,
-                'password': password,
-                'RECONNECT': 'FALSE'}
+        self._connection_data = {'userkey': userkey,
```

Comment:
This makes things strange. Storing and returning the same value doesn't look ideal. If you need this information and you cannot run the method, maybe we should refactor it to `_set_connection_data`, and use the inner value in all the code, and don't have 2 different usages

Reply:
ok I will do that.